lib/groebner.gd

```diff
@@ -485,6 +485,8 @@ DeclareOperation("GroebnerBasis",
 DeclareOperation("GroebnerBasis",[IsPolynomialRingIdeal,IsMonomialOrdering]);
 DeclareGlobalFunction("GroebnerBasisNC");
 
+DeclareSynonym("GrobnerBasis",GroebnerBasis);
```

Comment:
If that would not be considered as a gratuitous abouse of the language feature, I'd be happy to do so.